src/Symfony/Component/Console/Command/Command.php

```diff
@@ -678,6 +681,11 @@ public function asXml($asDom = false)
         return $output->fetch();
     }
 
+    public function setInputBound($inputBound)
```

Comment:
its always suspicious to me when a bug fix introduces a new public method - also, I don't get what this does without reading twice or more the code itself. is it me? :)